docs/releasehistory.rst

```diff
@@ -7,6 +7,78 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
+  But, to allow for independent development of individual parameter types, each section (such as ``Bonds``, ``Angles``, etc) now has its own version as well (currently all 0.3).
+* All units are now stored in expressions with their corresponding values. For example, distances are now stored as ``1.526*angstrom``, instead of storing the unit separately in the section header.
+* The ``potential`` field for ``ProperTorsions`` and ``ImproperTorsions`` tags is no longer ``charmm``, but is rather ``k*(1+cos(periodicity*theta-phase))``.
+  It was pointed out to us that CHARMM-style torsions deviate from this formula when the periodicity of a torsion term is 0, and we do not intend to reproduce that behavior.
+* SMIRNOFF spec documentation has been updated with tables of keywords and their defaults for each parameter section and parameter type.
+  These tables will track the allowed keywords and default behavior as updated versions of individual parameter sections are released.
+
+Performance improvements
+""""""""""""""""""""""""
+
+* `PR #329 <https://github.com/openforcefield/openforcefield/pull/329>`_: Improved runtime for system creation in large topologies.
+
+
+New features
+""""""""""""
+
+* `PR #311 <https://github.com/openforcefield/openforcefield/pull/311>`_: Several new experimental functions.
+    * Adds ``convert_0_2_smirnoff_to_0_3``, which takes a SMIRNOFF 0.2-spec data dict, and updates it to 0.3.
+      This function is called automatically when creating a ``ForceField`` from a SMIRNOFF 0.2 spec OFFXML file.
+    * Adds ``convert_0_1_smirnoff_to_0_2``, which takes a SMIRNOFF 0.1-spec data dict, and updates it to 0.2.
+      This function is called automatically when creating a ``ForceField`` from a SMIRNOFF 0.1 spec OFFXML file.
+    * NOTE: The format of the "SMIRNOFF data dict" above is likely to change significantly in the future.
+      Users that require a stable serialized ForceField object should use the output of ``ForceField.to_string('XML')`` instead.
+    * Adds ``ParameterHandler`` and ``ParameterType`` ``add_cosmetic_attribute`` and ``delete_cosmetic_attribute`` functions.
+      Once created, cosmetic attributes can be accessed and modified as attributes of the underlying object (eg. ``ParameterType.my_cosmetic_attrib = 'blue'``)
+      These functions are experimental, and we are interested in feedback on how cosmetic attribute handling could be improved. (`See Issue #338 <https://github.com/openforcefield/openforcefield/issues/338>`_)
+      Note that if a new cosmetic attribute is added to an object without using these functions, it will not be recognized by the toolkit and will not be written out during serialization.
+    * Values for the top-level ``Author`` and ``Date`` tags are now kept during SMIRNOFF data I/O.
+      If multiple data sources containing these fields are read, the values are concatenated using "AND" as a separator.
+
+
+
+API-breaking changes
```

Comment:
Is it possible to have these functions described below have links that resolve to the API docs using [Sphinx cross-referencing](https://stackoverflow.com/questions/22700606/how-would-i-cross-reference-a-function-generated-by-autodoc-in-sphinx)?